client/core/core.go

```diff
@@ -2663,6 +2663,19 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 	return nil
 }
 
+// handleNotifyMsg is called when a notify notification is received.
+func handleNotifyMsg(c *Core, dc *dexConnection, msg *msgjson.Message) error {
+	var txt string
+	err := msg.Unmarshal(&txt)
+	if err != nil {
+		return fmt.Errorf("notify unmarshal error: %v", err)
+	}
+	txt = fmt.Sprintf("Message from DEX at %s: %s", dc.acct.host, txt)
```

Comment:
How about quoting the message and starting it on a new line?  i.e. `:\n\n\"%s\"\n`

The web ntfn widget eats the newlines, so the quotes are only helpful there.  Ideally there would just be newlines without the quotes for the console but that's OK.